examples/jetstream/consumers/main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/nats-io/nats.go"
+)
+
+/*
+This example shows implementation of consumers management.
+`js-consumer-management` command can be used to create, update or get information about a consumer.
+`-push-target` option can be used to configure a delivery subject for a push-based consumer. If not provided, the consumer will be created in push mode.
```

Comment:
Parameter description should reflect if `-push-target` not provided the consumer will be created in **pull** mode.